lib/set_up.rb

```diff
@@ -0,0 +1,36 @@
+class SetUp
+
+  def initialize(console)
+    @console = console
+  end
+
+  def get_player(type, position)
+    if type == 'human'
+      HumanPlayer.new(position, @console)
```

Comment:
This is an example of why it is unusual to pass in the same instance of console. The `HumanPlayer` and the `ComputerPlayer` have different things passed into them, and this is the only place in `SetUp` where uses console.